utils/syncthing/Makefile

```diff
@@ -1,12 +1,11 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=syncthing
```

Comment:
Why is this gone?

Reply:
I wanted to test builds via the backported CI but didn't get around to do so yet